js/src/admin/components/UserListPage.tsx

```diff
@@ -88,7 +91,17 @@ export default class UserListPage extends AdminPage {
    */
   content() {
     if (typeof this.pageData === 'undefined') {
-      this.loadPage(0);
+      const indexOfSearch = location.hash.indexOf('?') + 1;
+      const search = indexOfSearch >= 0 ? location.hash.substr(indexOfSearch) : '';
+
+      const params = new URLSearchParams(search);
```

Comment:
I wonder if it might make sense to store this info in params namespaced by a table ID. That way, we could support having multiple tables on the same page, and also keep this logic in the table component.

Alternatively, we could support both modes, but error if more than one table / paginated list tries to use non-namespaced-mode?